update_db.txt

```diff
@@ -1,2 +1,2 @@
-0 9,15,21 * * * ./update_campus.sh
-@weekly ./update_ctown.sh
+0 9,15,21 * * * cd /usr/src/app; /usr/local/bin/python update_campus.py >> /var/log/cron.log 2>&1
+@weekly cd /usr/src/app; /usr/local/bin/python update_ctown.py >> /var/log/cron.log 2>&1
```

Comment:
Change `@weekly` to `0 0 * * 0` (=every Sunday midnight) please

Reply:
oh wait we should adjust it for utc time like we do for the update campus job

Reply:
`0 5 * * 0` = 5AM UTC / 12AM EST Sundays